server/routes/service.js

```diff
@@ -1,3 +1,153 @@
-const transport = require('../routes/transport');
-const db = require('../db/db'); 
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+const jwt = require('jsonwebtoken');
+const db = require('../db/db');
 
+
+// Check if the user is already authenticated
+function checkAuthStatus(authCookie) {
+  jwt.verify(authCookie, process.env.ACCESS_TOKEN_SECRET, (err) => {
+    if (err) {
+      console.log(err);
+      return '403';
+    }
+  });
+};
+
+// Check user existense
+async function signInDbQuery(name, pw) {
+  let checkedReturn;
```

Comment:
This doesn't need to be defined here.  You can just return the values at lines 31, 33, and 40 without setting it equal to this `checkedReturn` value.